src/metadata.rs

```diff
@@ -9,55 +9,78 @@ use serde::{
 use std::{borrow::Cow, collections::HashMap, fmt, path::PathBuf, str::FromStr, time::SystemTime};
 use url::Url;
 
-/// Represents a unique package identifier in a registry.
-///
-/// This identifier is unique within a specific registry.
-#[derive(Debug, Clone, PartialEq, Eq, Hash, Serialize, Deserialize, Ord, PartialOrd)]
-#[serde(transparent)]
-pub struct PackageId(String);
-
-impl PackageId {
-    /// Creates a new package id.
-    pub fn new(s: impl Into<String>) -> Self {
-        s.into().into()
+/// Represents a component model identifier.
```

Comment:
Could you add a quick example of the component-model identifier syntax to the doc comment here so that people jumping into this can connect it with what they see in wit?

Reply:
I'll add that.